test/jobs/create-version-branch.js

```diff
@@ -4,6 +4,9 @@ const proxyquire = require('proxyquire').noCallThru()
 
 const dbs = require('../../lib/dbs')
 
+nock.disableNetConnect()
+nock.enableNetConnect('localhost')
```

Comment:
I wonder why we don't have this for all our tests, we actually never want to call GitHub by accident. Good catch 👍